src/post-polyfill.js

```diff
@@ -18,4 +18,8 @@
     });
   });
 
+  if (customElements && customElements.polyfillWrapFlushCallback) {
+    customElements.polyfillWrapFlushCallback(flush => HTMLImports.whenReady(flush));
```

Comment:
value: function polyfillWrapFlushCallback(outer) {
          this._documentConstructionObserver.disconnect();
          ....
 }

 _createClass(DocumentConstructionObserver, [{
        key: 'disconnect',
        value: function disconnect() {
          this._observer.disconnect();  //this._observer == undefined
        }
      }
sorry for the span, but I do not understand polyfillWrapFlushCallback can be called before documentReady?